docs/can-guides/contribute/including-new-modules.md

```diff
@@ -1,21 +1,81 @@
-@page guides/contributing/adding-ecosystem-modules New Package
+@page guides/contributing/new-package New Package
```

Comment:
the old slug creates ordering problems I'll check for broken links before I release